ffig/templates/rs.tmpl

```diff
@@ -0,0 +1,12 @@
+{% for class in classes -%}
+pub struct Tree {
```

Comment:
Could be {{class.name}} rather than Tree?

Reply:
done